core/envelope-sim/src/testFixtures/java/fr/sncf/osrd/envelope_sim/EnvelopeSimPathBuilder.java

```diff
@@ -0,0 +1,59 @@
+package fr.sncf.osrd.envelope_sim;
+
+import com.google.common.collect.Range;
+import com.google.common.collect.RangeMap;
+import com.google.common.collect.TreeRangeMap;
+import java.util.Map;
+
+public class EnvelopeSimPathBuilder {
+    public  static EnvelopeSimPath withElectricalProfiles1500() {
```

Comment:
```suggestion
    public static EnvelopeSimPath withElectricalProfiles1500() {
```

I suspect something is wrong with the auto checkstyle

Reply:
Maybe we should make an issue for that ?